dll/cpl/desk/lang/es-ES.rc

```diff
@@ -276,5 +276,5 @@ BEGIN
     IDS_DISPLAY_SETTINGS "Configuración del monitor"
     IDS_APPLY_FAILED "Error al aplicar la nueva configuración..."
     IDS_APPLY_NEEDS_RESTART "Necesita reiniciar el equipo para aplicar los cambios."
-    IDS_TIMEOUTTEXT "Regresar a la última configuraciín en %d segundos."
+    IDS_TIMEOUTTEXT "Regresar a la última configuración en %d segundos."
```

Comment:
I know you didn't write this, but in spain we normally use "Volver" instead of "Regresar"...

Reply:
In fact I translated this in my previous pull request. The term "back" has the two meanings in spanish, so I choosed the most generic one (for me), but "Volver" seems to be better as uses less chars than "Regresar"